src/stores/CrowdsalePageStore.js

```diff
@@ -10,12 +10,52 @@ class CrowdsalePageStore {
   @observable rate;
   @observable tokensSold;
   @observable tokenAmountOf;
-  @observable startBlock
-  @observable endDate;
+  @observable tiers = []
+  @observable ticks = []
 
   @action setProperty = (property, value) => {
     this[property] = value
   }
+
+  @action setTier = (tier) => {
```

Comment:
The name of this method is misleading, it should be something like `addTier`, otherwise it seems idempotent.

Reply:
done, thanks!